src/storage/store/account/onchain_event_store.rs

```diff
@@ -590,6 +590,45 @@ impl OnchainEventStore {
         Ok((fids, next_page_token))
     }
 
+    pub fn get_highest_fid(&self) -> Result<Option<u64>, OnchainEventStorageError> {
+        let start_prefix = make_onchain_event_type_prefix(OnChainEventType::EventTypeIdRegister);
+        let stop_prefix = increment_vec_u8(&start_prefix);
```

Comment:
Do we need to decrement because we're paging in reverse? Or does the pagination logic automatically handle flipping these? Either way, maybe worth a comment.

Reply:
Yes, for paging in reverse. I'll add a comment